capellambse/helpers.py

```diff
@@ -326,7 +332,7 @@ def word_wrap(text: str, width: float | int) -> list[str]:
     """Perform word wrapping for proportional fonts.
 
     Whitespace at the beginning of input lines is preserved, but other
-    whitespace is collapsed to single spaces.
+    whitespace is collapsed to single spaces. Keep words as a whole.
```

Comment:
This description doesn't match the new implementation: whitespace at the beginning of lines currently is completely discarded. Please either update the description or fix the implementation. (I assume the implementation is correct, since you also updated the tests to reflect the new behavior.)